cookbook/import_export/create-connector.rst

```diff
@@ -3,8 +3,8 @@ How to Create a New Connector
 
 We'll implement here a very minimalist Connector, it will do nothing but allow us to understand the main concepts and the overall architecture.
 
-Create our Connector
---------------------
+Create your Bundle
```

Comment:
no pb, main point is also to be consistent with all other cookbook, we'll need to "standardize" at some point :)